advanced-usage/sealed-mocking.md

```diff
@@ -4,119 +4,53 @@ page_title: Sealed Mocking | JustMock Documentation
 description: Sealed Mocking
 previous_url: /advanced-usage-sealed-mocking.html
 slug: justmock/advanced-usage/sealed-mocking
-tags: sealed,mocking
+tags: sealed,mocking, mock, class
 published: True
 position: 3
 ---
 
-# Sealed Mocking
+# Mock Sealed Classes
 
-Sealed mocking is one of the advanced features supported in __Telerik® JustMock__. It allows you to fake sealed classes and calls to their methods/properties, set expectations and verify results using the AAA principle. Faking sealed classes and calls to their methods/properties doesn't affect the way you write your tests, i.e. the same syntax is used for mocking non sealed classes.
+This functionality allows you to fake sealed classes and calls to their members, set expectations and verify results using the [AAA]({%slug justmock/basic-usage/arrange-act-assert%}) principle. Mocking sealed classes and calls to their methods/properties doesn't affect the way you write your tests, i.e. the same syntax is used for mocking non-sealed classes.
 
 > This feature is available only in the commercial version of Telerik JustMock. Refer to [this]({%slug justmock/getting-started/commercial-vs-free-version%}) topic to learn more about the differences between the commercial and free versions.
```

Comment:
There are two consecutive note sections, which is a bit confusing. Please reduce them to just one.